From: Ard Biesheuvel <ard.biesheuvel@linaro.org>
Date: Mon, 11 Jul 2016 21:00:46 +0200
Subject: efi/arm*: esrt: Add missing call to efi_esrt_init()

Git-commit: 2ead3084e3fc37d42f379cca8753b458d8f9ba25
Patch-mainline: v4.9-rc1
References: fate#320512

ESRT support is built by default for all architectures that define
CONFIG_EFI. However, this support was not wired up yet for ARM/arm64,
since efi_esrt_init() was never called. So add the missing call.

Signed-off-by: Ard Biesheuvel <ard.biesheuvel@linaro.org>
Cc: Leif Lindholm <leif.lindholm@linaro.org>
Cc: Mark Rutland <mark.rutland@arm.com>
Cc: Peter Jones <pjones@redhat.com>
Signed-off-by: Matt Fleming <matt@codeblueprint.co.uk>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/firmware/efi/arm-init.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/firmware/efi/arm-init.c
+++ b/drivers/firmware/efi/arm-init.c
@@ -271,6 +271,7 @@ void __init efi_init_fdt(void *fdt)
 
 	reserve_regions();
 	efi_memattr_init();
+	efi_esrt_init();
 	efi_memmap_unmap();
 	memblock_reserve(params.mmap & PAGE_MASK,
 			 PAGE_ALIGN(params.mmap_size +
